src/ucs/debug/debug.c

```diff
@@ -1133,43 +1122,28 @@ static int ucs_debug_is_error_signal(int signum)
     return result;
 }
 
-sighandler_t signal(int signum, sighandler_t handler)
+sighandler_t ucs_debug_signal(int signum, sighandler_t handler)
```

Comment:
can we fully move this function to libucs_signal.c, and not use it from ucs?
it should be called something like "ucs_orig_signal()"